workflow-templates/add-issue-to-a-board.yml

```diff
@@ -0,0 +1,17 @@
+# Run the workflow that adds newly created issues to a specific
+# GitHub Project
+
+name: Add newly created issues to the project board
+
+on:
+  issues:
+    types: [opened]
```

Comment:
Took me an embarrassingly long time to read this as "past tense of open" and not "a misspelling of openedx" 😛

Reply:
Lol, same thing happened to Carlos on a different github action we were writing together.